pkg/display/display.go

```diff
@@ -61,7 +65,11 @@ func (d *Display) Bypass() io.Writer {
 }
 
 func (d *Display) Close() {
-	close(d.close)
+	select {
+	case <-d.close:
+	default:
+		close(d.close)
+	}
```

Comment:
I think this has a race condition. If you enter the default case but another goroutine closes `d.close` before you do, you'll panic.

Reply:
I'll trade you a close for a ctx, cancel?